zodiac/sources/ZodiacSource.py

```diff
@@ -0,0 +1,29 @@
+from typing import Union
+
+import discord
+from discord import app_commands
+
+from pie.i18n import TranslationContext
+
+from ..ZodiacSign import ZodiacSign
+
+
+class ZodiacSource:

```

Comment:
```suggestion
class ZodiacSource:
   """Base class for Zodiac sources. The method `get` must be overwritten.
   Any subclass in the sources folder is considered a source and added to
   sources variable in __init__.py.
   
   :param sign: Zodiac sign to generate embed for
   :param utx: Translation context
   """
```

I think the docs might be useful here.